Reflection_Engine/Compute/LoadAllAssemblies.cs

```diff
@@ -75,11 +75,12 @@ public static List<Assembly> LoadAllAssemblies(string folder = "", string regexF
                 SearchOption searchOption = parseSubfolders ? SearchOption.AllDirectories : SearchOption.TopDirectoryOnly;
                 foreach (string file in Directory.GetFiles(folder, "*.dll", searchOption))
                 {
-                    string[] parts = file.Split(new char[] { '.', '\\' });
+                    string[] parts = file.Split(new char[] { '\\' });
                     if (parts.Length < 2)
                         continue;
 
-                    string name = parts[parts.Length - 2];
+                    string name = parts[parts.Length - 1];
```

Comment:
The check on `parts.Length` was only necessary when the file extension was split into its own part. Now that this doesn't happen, the check should be removed. 

While you are at it, I also suggest to used `parts.Last()` instead of `parts[parts.Length - 1]` as it is cleaner.

Reply:
Yeah you are right - I cleaned it up properly this time by using string `Path.GetFileNameWithoutExtension()` method 👍